src/components/RowsPerPageToggleButton.js

```diff
@@ -1,20 +1,22 @@
-import React from 'react';
+import React, { useState } from 'react';
 import "../css/RowsPerPageToggleButton.css"
 
 
 export function RowsPerPageToggleButton( props ) {
 
+	const [row10active, setRow10Active] = useState(true)
```

Comment:
We have this state stored in ResearchPage.js right?
So I think we can just pass it in as a prop, and adjust the CSS based on that?

Reply:
Changed, did see there is this state lol mb